app/modules/entities/components/layouts/entity_layout_actions.svelte

```diff
@@ -72,6 +73,21 @@
     />
   </li>
 {:else}
+  {#if isIsbnEntityUri(uri)}
```

Comment:
It's an old issue (https://github.com/inventaire/inventaire/issues/192) but since removing isbn's uris is still a possibility, what about directly checking if isbn exists ? (Since server's resolver is using this prop value, while uri is an indirection)
```suggestion
  {#if claims['wdt:P212']}
```

Reply:
done in c88a487